packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/ParagraphScopeHandler.ts

```diff
@@ -7,11 +7,11 @@ import {
 } from "@cursorless/common";
 import { Direction, ScopeType } from "@cursorless/common";
 import { ParagraphTarget } from "../../targets";
-import BaseScopeHandler from "./BaseScopeHandler";
+import { BaseScopeHandler } from "./BaseScopeHandler";
 import { fitRangeToLineContent } from "./LineScopeHandler";
 import { TargetScope } from "./scope.types";
 
-export default class TokenScopeHandler extends BaseScopeHandler {
+export class ParagraphScopeHandler extends BaseScopeHandler {
```

Comment:
Whoops nice catch 😅

Reply:
Well, the compiler caught it for me. :-)